feature/home/src/main/java/com/teamoffroad/feature/home/presentation/component/download/DownloadImage.kt

```diff
@@ -49,15 +47,28 @@ private suspend fun loadImage(
     imageUrl: String
 ): DownloadResult<Bitmap> {
     return withContext(Dispatchers.IO) {
-        val imageLoader = ImageLoader(context)
+        val imageLoader = ImageLoader.Builder(context)
+            .components {
+                add(SvgDecoder.Factory())
+            }
+            .build()
+
         val request = ImageRequest.Builder(context)
             .data(imageUrl)
             .build()
 
         when (val result = imageLoader.execute(request)) {
-            is SuccessResult -> DownloadResult.Success(result.drawable.toBitmap())
-            is ErrorResult -> DownloadResult.Error("이미지 다운로드 실패: ${result.throwable.message}")
-            else -> DownloadResult.Error("이미지 다운로드 실패: 요청 실패")
+            is SuccessResult -> {
+                val drawable = result.drawable
+                val bitmap = (drawable as? BitmapDrawable)?.bitmap
```

Comment:
```suggestion
                val bitmap = (result.drawable as? BitmapDrawable)?.bitmap
```
c: 일케 가능한가요? 이쪽이 더 가독성이 좋아보이는데 킁

Reply:
오오 수정하겠습니다.